include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -7269,6 +7269,10 @@ namespace internal
   VectorizedArrayType *
   check_vector_access_inplace(const EvaluatorType &fe_eval, VectorType &vector)
   {
+    // for user-defined macro cells this functionality is not supported
```

Comment:
Can we use what you suggest in the name of the PR,
```suggestion
    // for user-defined cell batches this functionality is not supported
```